charts/promtail/templates/service-extra.yaml

```diff
@@ -4,6 +4,7 @@ apiVersion: v1
 kind: Service
 metadata:
   name: {{ include "promtail.fullname" $ }}-{{ $key | lower }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
I think it's best to define a variable for the `.Release.Namespace` before the `range` loop.
https://helm.sh/docs/chart_template_guide/variables/

Reply:
Fixed it.